src/Views/scstormrunoff.html

```diff
@@ -378,12 +378,290 @@ <h4 class="modal-title">New {{flowType.displayName}} Segment</h4>
+                                            <input  type="number" name="w1" ng-model="vm.w1" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+                                            <label for="w2">W<sub>2</sub> (feet):<span style="color: #ffcc00">*</span></label><span ng-if="SyntheticUrbanHydrograph.w2.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than 0</span>
+                                            <input  type="number" name="w2" ng-model="vm.w2" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+                                            <label for="side_slope_z">Side Slope Z:<span style="color: #ffcc00">*</span></label><span ng-if="SyntheticUrbanHydrograph.side_slope_z.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than 0</span>
+                                            <input  type="number" name="side_slope_z" ng-model="vm.side_slope_z" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+                                            <label for="bottom_slope">Bottom Slope S<sub>O</sub> (%):<span style="color: #ffcc00">*</span></label><span ng-if="SyntheticUrbanHydrograph.bottom_slope.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than 0</span>
+                                            <input  type="number" name="bottom_slope" ng-model="vm.bottom_slope" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+                                        </div>
+
+                                        
+                                        <!-- TODO: highlight input box in Elevation column if values are not within range -->
+                                        <div ng-if="vm.pondOption == 2">
+                                            <div class="form-row">
+                                                <div class="form-column-left">
+                                                    <label>Elevation (ft-MSL) (values should be between {{vm.pond_bottom_elev}} and {{vm.pond_bottom_elev + 10}})<span style="color: #ffcc00">*</span></label>
+                                                </div>
+                                                <div class="form-column-right">
+                                                    <label for = "elevation">Surface Area (square feet)<span style="color: #ffcc00">*</span></label>
+                                                </div>
+                                            </div>
+                                            <div class="form-row">
+                                                <div class="form-column-left">
+                                                    <input type="number" value="{{vm.pond_bottom_elev}}" ng-model="vm.Elev_Area[0][0]" readonly/>
```

Comment:
Is there a way to signify that this input field is disabled? Maybe add a `disabled` attribute? So the user knows this one is different and not editable.